lib/Doctrine/DBAL/Driver/PDOStatement.php

```diff
@@ -274,10 +271,13 @@ private function convertParamType(int $type): int
     {
         if (! isset(self::PARAM_TYPE_MAP[$type])) {
             // TODO: next major: throw an exception
-            @trigger_error(sprintf(
-                'Using a PDO parameter type (%d given) is deprecated and will cause an error in Doctrine DBAL 3.0',
+            Deprecation::trigger(
+                'doctrine/dbal',
+                'https://github.com/doctrine/dbal/pull/3088',
+                'Using a PDO parameter type (%d given) is deprecated, ' .
+                'use \Doctrine\DBAL\Types\Types constants instead.',
```

Comment:
One could use `Types::class` instead.

Reply:
Yeah weird feeling with that one was that "Types" might have not been read as a class by users, where in the other cases with the ::method its more obvious.